.github/workflows/zulip-ci.yml

```diff
@@ -80,7 +80,7 @@ jobs:
         uses: actions/cache@v4
         with:
           path: /srv/zulip-venv-cache
-          key: v1-venv-${{ matrix.os }}-${{ hashFiles('requirements/dev.txt') }}
+          key: v1-venv-${{ matrix.os }}-${{ hashFiles('uv.lock') }}
```

Comment:
TODO: [Set up caching correctly](https://docs.astral.sh/uv/concepts/cache/#caching-in-continuous-integration)